src/olympia/devhub/forms.py

```diff
@@ -878,3 +879,47 @@ class DistributionChoiceForm(happyforms.Form):
 class AgreementForm(happyforms.Form):
     distribution_agreement = forms.BooleanField()
     review_policy = forms.BooleanField()
+
+
+class SingleCategoryForm(happyforms.Form):
+    category = forms.ChoiceField(widget=forms.RadioSelect)
+
+    def __init__(self, *args, **kw):
+        self.addon = kw.pop('addon')
+        self.request = kw.pop('request', None)
+        super(SingleCategoryForm, self).__init__(*args, **kw)
+
+        form = self.fields['category']
+        # Hack because we know this is only used for Static Themes that only
+        # support Firefox.  Hoping to unify per-app categories in the meantime.
+        app = amo.FIREFOX
+        sorted_cats = sorted(CATEGORIES[app.id][self.addon.type].items(),
+                             key=lambda (slug, cat): slug)
+        choices = [
+            (c.id, c.name) for _, c in sorted_cats]
+        form.choices = choices  # sorted(choices, key=lambda x: x.name)
+
+        initials = self.addon.app_categories.get(app.id, [])
+        if len(initials) > 0:
+            form.initial = initials[0]
+
+        # If this add-on is featured for this application, category changes are
+        # forbidden.
```

Comment:
Why? I've skimmed through the docs for the static themes and could not find a reference to this.

Reply:
Oh, that's something I pulled over from the multiple category form https://github.com/mozilla/addons-server/blob/master/src/olympia/addons/forms.py#L279 - no-one's mentioned it either way so I thought I'd keep the same behavior as extensions.

Reply:
Ah, I had never noticed this. Not a big deal, let's keep it for now, I was just wondering.